### ISSUE TITLE
Doc is too vague for the meaning of x and y in /ostree/boot.x.y/

### ISSUE THREAD

**Issue by gatispaeglis** [projectatomic/atomic-site#183]
I found two resources: 
1) https://access.redhat.com/articles/rhel-atomic-getting-started explanation of that path is very vague
2) https://wiki.gnome.org/Projects/OSTree/DeploymentModel2 is really out-of-date, last edited in 2013

Reading from the above resources does not give a clear answer to what is represented by x and y in /ostree/boot.x.y/ path.

**Comment by gatispaeglis** [comment]
The current docs from redhat site state:
"These are the actual directories that contain the file system that will be booted. These directories consist of hard links into the repo directory to minimize disk space consumption."

And this is my understanding when looking at the source code:
boot.x.y implements an optimization to avoid writing bootloader configurations when possible. If deployment count does not change and deployment bootconfigs are equal (see deployment_bootconfigs_equal) then it goes down the optimized path where:
x - does not change (because of the paths in bootloader configs)
y - changes 0 <-> 1

When the above condition is false then it goes the other path, and updates bootloader configurations accordingly. 
x - changes 0 <-> 1
y - does not change

x = [0 , 1]
y = [0 , 1]

In summary:

I don't see what the above docs mean by "minimize disk space consumption." :)

**Comment by jberkus** [comment]
Which doc are we talking about here?  That is, which doc on the Atomic Site?

**Comment by gatispaeglis** [comment]
I think I reported this to the wrong place :) The issue was on the RedHat site (link avove), but I see that they have removed that sentence already.
Atomic is Redhat project, that is probably how I ended up reporting it here..
